ruby/vampires.rb

```diff
@@ -30,10 +30,6 @@
       insurance = false
     end
```

Comment:
For conditionals like lines 21 - 31 where the value collected is either `true` or `false` you could consider assigning the variable to a default prior to the condition.  Then you can write one less condition.  It is cleaner logically and saves on runtime.  Also don't reassign a string var to a boolean.  have seperate values for the input and the collected variable.

```
wants_garlicbread = false

p "would yo like some garlicbread? (y/n)"

if gets.chomp == y
     wants_garlicbread = true
end
```